trlx/trainer/accelerate_ppo_trainer.py

```diff
@@ -435,7 +437,7 @@ def make_experience(self, num_rollouts: int = 1024, iter_count: int = 0):  # noq
                 start = prompt_tensors.shape[1] - 1
 
             log_ratio = (logprobs - ref_logprobs) * attention_mask[:, :-1]
-            self.mean_kl = (log_ratio.exp() - 1 - log_ratio).mean().to(device)
+            self.mean_kl += (log_ratio.exp() - 1 - log_ratio).sum().to(device)
```

Comment:
why are we changing the accelerate implementation? Shouldn't this be its own PR?